packages/wallet/src/redux/funding/indirect-funding/player-a/state.ts

```diff
@@ -0,0 +1,76 @@
+import { Properties } from '../../../utils';
+import { PlayerIndex } from '../../../types';
+
+export const WAIT_FOR_APPROVAL = 'WAIT_FOR_APPROVAL';
+export const WAIT_FOR_PRE_FUND_SETUP_1 = 'WAIT_FOR_PRE_FUND_SETUP_1';
```

Comment:
Should this be `WAIT_FOR_PRE_FUND_SETUP`? The fact that it is the first prefund state is implied by the `player` field.

Reply:
I kind of agree with @kerzhner here, but after that change, it starts to seem silly to have separate states for player A and player B -- they share exactly the same code, except they have a different value in the `playerIndex` property.

Reply:
We're going to stick with the code as is. It's more complicated than it needs to be, but it's already written, and tom seems to want it to be explicit.